### PR TITLE
Repositories: Added librepilot overlay

### PR DESCRIPTION
**Pull request by paul-jewell**
I have created an overlay for the LibrePilot flight software. This pull request adds the details to the repositories file.

### CODE REVIEW

**Comment by paul-jewell** [comment]
I changed the repository name to lower case, then uploaded the changes. I then changed the name in the profiles/repo-name file and pushed that, but the checks failed before I managed that. How can I restart the checks? They should now show a consistent condition.

File: files/overlays/repositories.xml [hunk 1]
```diff
@@ -2582,7 +2582,7 @@ FIN
     </owner>
     <source type="git">https://github.com/paul-jewell/librepilot-overlay.git</source>
     <source type="git">git://github.com/paul-jewell/librepilot-overlay.git</source>
```
Comment (mgorny) [inline files/overlays/repositories.xml]: Please do not include `git://` or any other insecure fallbacks.

File: files/overlays/repositories.xml [hunk 2]
```diff
@@ -2582,7 +2582,7 @@ FIN
...
-    <source type="git">git@github.com:paul-jewell/librepilot-overlay.git</source>
+    <source type="git">git+ssh://git@github.com:paul-jewell/librepilot-overlay.git</source>
```
Comment (mgorny) [inline files/overlays/repositories.xml]: And please follow suit with other overlays hosted on GitHub wrt URL form.

File: files/overlays/repositories.xml (hunk 1 above)
Comment (paul-jewell) [reply files/overlays/repositories.xml]: I will remove this line. I do notice it is present in some of the other github hosted repositories though (eg libressl)?

File: files/overlays/repositories.xml (hunk 2 above)
Comment (paul-jewell) [reply files/overlays/repositories.xml]: No problem - Just for clarification, you are referring to the colon between github.com and paul-jewell? I'll change it to a '/' as shown in the template.

**Comment by paul-jewell** [comment]
Hi Michael,

I know you are busy. I will have a look at cleaning it up and issue some pull requests, if that would help.

Best regards,
Paul

**Comment by mgorny** [comment]
If you can look into updating them to resemble templates better, I'd appreciate that. However, don't do it manually — if you can't script it yourself, I'll eventually get to it ;-).

**Comment by paul-jewell** [comment]
I like a challenge :). I'll look at scripting it!

**Pull request closed without merging**